src/components/common/view/BigGoodDollar.js

```diff
@@ -2,17 +2,61 @@
 import React from 'react'
 import { weiToMask } from '../../../lib/wallet/utils'
 import BigNumber from './BigNumber'
+import Text from './Text'
 
 type Props = { number: any, props?: {} }
 
+const GOOD_SIGN_SIZE = 18
+const DIFF_FACTOR = 0.5
+const DOLLAR_SIGN_SIZE = GOOD_SIGN_SIZE - DIFF_FACTOR
+
 /**
  * Receives wei and shows as G$ using BigNumber component
  * @param {Props} params
  * @param {Number} params.number
  * @returns {React.Node}
  */
-const BigGoodDollar = ({ number, ...props }: Props) => (
-  <BigNumber number={number === undefined ? '-.--' : weiToMask(number)} unit={'G$'} {...props} />
-)
+const BigGoodDollar = ({ number, formatter, ...props }: Props) => {
+  const formattedNumber = formatter && formatter(number)
```

Comment:
I think that something like this would be more simple to understand

```
  const numberFormatter = formatter || weiToMask
  number = number === undefined ? '-.--' : numberFormatter(number)
```

Reply:
indeed :tada:

thanks for the suggestion